### ISSUE TITLE
Measure actual ND filter transmission functions

### ISSUE THREAD

**Issue by awhoward** [Keck-DataReductionPipelines/KPF-Pipeline#460]
Because the wedge angles in the ND filters are inhomogeneous and don't meet our specifications, the optical beams that feed the fibers on the Cal Bench are steered off of the fiber centers.  The amount of steering depends on the combination of the two ND filters chosen.  The result of the beam steering is that some combinations of ND filters result in better fiber coupling than others.  For some fiber combinations, I think we miss the beam entirely.

We have a plan to fix this with wedged filters that are more uniform, but it is months from being implemented.  In the meantime, we need a way to estimate the expected amount of simultaneous cal light for an exposure of a given duration and a set of chosen ND filters.

The goal of this Issue is to go through every combination of ND filters using the LFC and using the etalon as sources and determine the relative flux.  (We're doing this for the LFC and etalon because the problem is exacerbated for sources that are carried in single-mode fibers.  Their beam sizes are smaller.)  

In other words, does going from the OD1.0 to OD2.0 filter result in a 10X reduction in the flux or some other amount?  (Spoiler: it will be another amount.)

These relative flux measurements can be used to determine a set of "effective optical densities" for the filters.  These combinations are the only set that we'll be able to select from for on-sky observations in the next few months.
